src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -59,15 +77,17 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
         IStakeRegistry _stakeRegistry,
         IBLSPubkeyRegistry _blsPubkeyRegistry,
         IIndexRegistry _indexRegistry
-    ) {
+    ) EIP712("AVSRegistryCoordinator", "v0.0.1") {
         slasher = _slasher;
         serviceManager = _serviceManager;
         stakeRegistry = _stakeRegistry;
         blsPubkeyRegistry = _blsPubkeyRegistry;
         indexRegistry = _indexRegistry;
     }
 
-    function initialize(OperatorSetParam[] memory _operatorSetParams) external initializer {
+    function initialize(address _churner, OperatorSetParam[] memory _operatorSetParams) external initializer {
+        // set the churner
+        churner = _churner;
```

Comment:
should presumably have an event associated to this

Reply:
added